src/tribler-core/tribler_core/modules/resource_monitor.py

```diff
@@ -167,14 +166,13 @@ def write_resource_logs(self, time_seconds):
         if not self.memory_data or not self.cpu_data:
             return
         with self.resource_log_file.open(mode="a+") as output_file:
-            output_file.write(u"%s, %s, %s\n" % (time_seconds,
-                                                self.memory_data[len(self.memory_data)-1][1],
-                                                self.cpu_data[len(self.cpu_data)-1][1]))
+            output_file.write(f"{time_seconds}, {self.memory_data[len(self.memory_data)-1][1]}, "
+                                f"{self.cpu_data[len(self.cpu_data)-1][1]}\n")
 
     def reset_resource_logs(self):
         resource_dir = self.resource_log_file.parent
```

Comment:
Make `resource_dir` a Path object?

Reply:
In this case it's unnecessary as `Path.parent` returns a `Path` object.
https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.parent